doc/cascadia/SettingsSchema.md

```diff
@@ -35,6 +35,10 @@ Properties listed below are specific to each unique profile.
 | `startingDirectory` | _Required_ | String | `%USERPROFILE%` | The directory the shell starts in when it is loaded. |
 | `useAcrylic` | _Required_ | Boolean | `false` | When set to `true`, the window will have an acrylic background. When set to `false`, the window will have a plain, untextured background. |
 | `background` | Optional | String | | Sets the background color of the profile. Overrides `background` set in color scheme if `colorscheme` is set. Uses hex color format: `"#rrggbb"`. |
+| `backgroundImage` | Optional | String | | Sets the file location of the Image to draw over the window background. |
+| `backgroundImageAlignment` | Optional | String | `center` | Sets how the background image aligns to the boundaries of the window. Possible values: `"center"`, `"left"`, `"top"`, `"right"`, `"bottom"`, `"topLeft"`, `"topRight"`, `"bottomLeft"`, `"bottomRight"` |

```

Comment:
Can you list out what the default behavior will be if you don't set the alignment/opacity/stretch given they're optional?

Reply:
I believe that's what column 4 is for, but I'm not _sure_

Reply:
Correct, column 4 ( **Default** ) lists the default settings of `center`, `1.0`, and `uniformToFill` respectively. Default values are listed for other optional settings in the **Globals** category as well.